### PR TITLE
Idle timeout

### PR DESCRIPTION
**Pull request by sedroche**
Jira: https://issues.redhat.com/browse/OSD-6565

Changing the idle timeout to 1800 from 600 on rh-api and default-router loadbalancers.

### CODE REVIEW

File: pkg/controller/utils/conditions.go [hunk 1]
```diff
@@ -50,6 +50,7 @@ func SetAPISchemeCondition(
 					Reason:             reason,
 					Message:            message,
 					LastTransitionTime: now,
+					LastProbeTime:      now,
```
Comment (sedroche) [inline pkg/controller/utils/conditions.go:53]: This was error-ing in the logs. `LastProbeTime` needs a  value set

Comment (csheremeta) [reply pkg/controller/utils/conditions.go:53]: this sounds like it will fix https://issues.redhat.com/browse/OSD-6789. I'll test this today.

**Comment by dustman9000** [comment]
My only concern with opening up the idle timeout from 10 to 30 minutes is that it has the potential to open up the api endpoint to a slow http request DoS attack. Can we just reduce the backplane timeout to 10m instead?

**Comment by csheremeta** [comment]
I tested this code today to ensure that it fixes both OSD-6565 and OSD-6789 (which it does). I also ran e2e tests against it, and it did not cause any failures.

**Comment by sedroche** [comment]
> My only concern with opening up the idle timeout from 10 to 30 minutes is that it has the potential to open up the api endpoint to a slow http request DoS attack. Can we just reduce the backplane timeout to 10m instead?

@dustman9000 For the rh-api endpoint, that is ip whitelisted. It can only be connected to from the bastions or hives. So its just SRE using it which will mean fewer connections

**Review by fahlmant — APPROVED** [review]
/lgtm

**Pull request merged**